.prettierrc.yml

```diff
@@ -0,0 +1,14 @@
+trailingComma: all
+printWidth: 80
+singleQuote: true
+arrowParens: avoid
+tabWidth: 4
+bracketSpacing: false
```

Comment:
Also not a huge fan of this myself - but if everyone else prefers it I will subside.

If nothing else, I think having spaces around brackets in object literals makes sense because we already have/enforce spaces around brackets in import statements, single line function bodies, etc.